sources/packages/web/src/services/http/dto/ApplicationException.dto.ts

```diff
@@ -9,8 +10,10 @@ export interface CreateApplicationExceptionAPIInDTO {
   exceptionRequests: ApplicationExceptionRequestAPIInDTO[];
 }
 
-export interface UpdateApplicationExceptionAPIInDTO {
+export class UpdateApplicationExceptionAPIInDTO {
+  @Expose()
```

Comment:
why we need this explicitly mentioned?

Reply:
Same from https://github.com/bcgov/SIMS/pull/2045#discussion_r1242859860